shared/common-adapters/name-with-icon.js

```diff
@@ -72,7 +72,7 @@ const NameWithIconVertical = (props: Props) => {
         {!!props.username && (
           <ConnectedUsernames
             type={adapterProps.titleType}
-            containerStyle={styles.vUsernameContainerStyle}
+            containerStyle={isMobile ? undefined : styles.vUsernameContainerStyle}
```

Comment:
use new `platformStyle` thing?

Reply:
`styles.vUsernameContainerStyle` is one of those objects. there are none that apply to mobile, which was causing an exception. I'll try to fix the root cause

Reply:
ah, got it

Reply:
Fixed by `isArray` fix below?

Reply:
Nah, the fix is to change `Usernames` to work with the new style objects. I started on that but the work started ballooning into other components, so I ticketed.